src/modules/holder/pages/Home/Home.tsx

```diff
@@ -41,21 +40,49 @@ export const Home: FC = () => {
       </>
     )
   }
+
+  const validTickets = data.filter((credentialItem) => {
+    const credentialSubject = (credentialItem as StoredW3CCredential)?.credentialSubject
+    if (Date.parse(credentialSubject?.eventDate) > Date.now()) {
+      return credentialItem
+    }
+  })
+
+  const expiredTickets = data.filter((credentialItem) => {
+    const credentialSubject = (credentialItem as StoredW3CCredential)?.credentialSubject
+    if (Date.parse(credentialSubject?.eventDate) <= Date.now()) {
+      return credentialItem
+    }
+  })
+
+  const getCredential = (credentialSubject: any, credentialItem: any) => {
+    const credential: Credential = {
+      title: credentialSubject?.eventName,
+      date: credentialSubject?.eventDate,
+      time: credentialSubject?.eventTime,
+      credentialId: (credentialItem as StoredW3CCredential)?.id,
+    }
+
+    return credential
+  }
+
+  const getTicketCards = (array: any, isActive: boolean) =>
+    array.map((credentialItem: any, index: any) => {
+      const credentialSubject = (credentialItem as StoredW3CCredential)?.credentialSubject
+
+      const credential = getCredential(credentialSubject, credentialItem)
+      return <TicketCard key={index} credential={credential} isActive={isActive} />
```

Comment:
react recommend to avoid using `index` for `key`  
is there some other uniqe value like id or name or something else?